tests/frontend/org/voltdb/newplanner/CalcitePlannerTestCase.java

```
@@ -60,10 +61,9 @@ protected CompiledPlan compileAdHocCalcitePlan(String sql,
         CompiledPlan cp = null;
         try {
             cp = m_aide.compileAdHocPlan(PlannerType.CALCITE, sql, inferPartitioning, forcedSP, detMode);
-            assertTrue(cp != null);
+            assertNotNull(cp);
         } catch (Exception ex) {
-            ex.printStackTrace();
-            fail();
+            fail(ex.getMessage());
```

Why not just let the exception bubble up and fail the test by having the test throw an exception. That way the entire stack trace is captured by junit.

---

Right, removed the try block.